script.js

```diff
@@ -1,66 +1,81 @@
 let currentValue = '';
 let currentOperator = '';
 let result = 0;
-
-function inputNum(value) {
-  currentValue += value;
-  document.getElementById('output').textContent = currentValue;
-}
-
+let displayScreen = document.getElementById('output');
 let operatorButtons = document.getElementsByClassName('operator');
```

Comment:
why are we using let here instead of const? are we changing these values somewhere?